backend/app/services/mongo/explore.py

```diff
@@ -2263,6 +2264,26 @@ async def fetch_flattened_tasks(
                 **return_columns,
                 "event_removed": "$events.removed",
                 "event_removal_reason": "$events.removal_reason",
+                "event_category_name": "$events.score_range.label",
+                "event_number_of_categories": {
+                    "$size": {
+                        "$ifNull": [
+                            "$events.event_definition.score_range_settings.categories",
+                            [],
+                        ]
+                    }
+                },
+                "event_category_index": {
```

Comment:
event_score_range_label_index

Reply:
Instead of doing this, directly return  events.event_definition.score_range_settings.categories as a list